### PR TITLE
⚠️ Decoupling IPAM from CAPM3

### PR DESCRIPTION
**Pull request by peppi-lotta**
These changes fix the issues produced by this [PR in CAPM3](https://github.com/metal3-io/ip-address-manager/pull/427)

### CODE REVIEW

File: config/default/kustomization.yaml [hunk 1]
```diff
@@ -3,5 +3,4 @@ commonLabels:
   cluster.x-k8s.io/provider: "infrastructure-metal3"
 
 resources:
-- capm3
-- ../ipam
\ No newline at end of file
+- capm3
```
Comment (tuminoid) [inline config/default/kustomization.yaml:6]: Line feed missing.

File: config/ipam/image_patch.yaml [hunk 2]
```diff
@@ -2,7 +2,7 @@ apiVersion: apps/v1
 kind: Deployment
 metadata:
   name: ipam-controller-manager
-  namespace: capm3-system
+  namespace: ipam-provider-m3-system
```
Comment (tuminoid) [inline config/ipam/image_patch.yaml:5]: That is one weird name. Is there reasoning behind that? I expected to see simple `ipam-system` or similar

**Review by tuminoid — COMMENTED** [review]
Nits.

BTW, the link in PR summary links to a check in a PR, not to PR itself.

File: config/ipam/image_patch.yaml (hunk 2 above)
Comment (lentzi90) [reply config/ipam/image_patch.yaml:5]: I agree. Suggested `ipam-metal3-system` in the IPAM PR

**Comment by peppi-lotta** [comment]
https://github.com/metal3-io/cluster-api-provider-metal3/pull/1993
This PR is doing the same thing as this. Closing.

**Pull request closed without merging**